Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -33,6 +34,16 @@ const SubmissionDetails = () => {
     } = useSubmissionsDetails();
     const { actions: { setPageTitle } } = usePageTitles();
     const { state: { user: { permissions: { canAccessAdministration } } } } = useAuth();
+    const {
+        state:
+            { downloadErrorMessage },
+        actions:
+            {
+                downloadProblemSubmissionFile,
+                setDownloadErrorMessage,
+            },
+    } =
+        useSubmissionsDetails();
```

Comment:
Place useSubmissionsDetails on the upper row.

Reply:
Done.